azure-functions/acbs-function/api.api-test.js

```diff
@@ -8,15 +8,15 @@ const {
 
 describe('get function', () => {
```

Comment:
This isn't quite what I meant in my comment, I was more thinking something like this:
```
describe('get function', () => {
  it('should return an api response object for valid requests, async () => {
    const response = await getFacility('1');
    expect(response).toHaveProperty('status');
    expect(response).toHaveProperty('data');
  });

  it('should return an api response object for invalid requests', async () => {
    const response = await getFacility();
    expect(response).toHaveProperty('status'); 
    expect(response).toHaveProperty('data');
  });
});
```
It's only subtly different, so I'll leave it up to you whether or not to change it. :)